pkgs/development/tools/sq/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, buildGoModule, fetchFromGitHub, installShellFiles }:
+
+buildGoModule rec {
+  pname = "sq";
+  version = "0.15.4";
+
+  src = fetchFromGitHub {
+    owner = "neilotoole";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-4CINLOHUVXQ+4e5I1fMqog6LubMm8RnbFmeuBOwALaw=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  vendorSha256 = "sha256-lNpWXKtnzwySzinNPxAKuaLqweWuS6zz8s2W4xXWlqM=";
+
+  # Some tests violates sandbox constraints.
+  doCheck = false;
```

Comment:
Why this though?

Reply:
`sq` seems to expect ability to use network to talk with PostgreSQL or something and it seems to fail. I have not been able to investigate why, plus, tests are quite long (even on a fairly fast machine) as they are comprehensive.
If someone feels comfortable enough to cherry pick the right subset of tests, I think it would be the best solution.

Reply:
Well I have no idea of sql.

Reply:
You could add a testVersion invocation, if the version is provided (through ldflags):
```nix
  passthru.tests = {
    version = testVersion { package = sq; };
  };
```

Reply:
Currently the version results in `v0.0.0-dev` (from defaultVersion) thus breaking the `tests.version`.
Version should be provided through `ldflags` (extracted from `$src/.goreleaser.yml`).
```
  ldflags = [
    "-s" "-w" "-X github.com/neilotoole/sq/cli/buildinfo.Version=${version}"
  ];
```

Reply:
Addressed.